src/mmw/js/src/draw/views.js

```diff
@@ -172,12 +172,18 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 });
 
 function changeOutlineLayer(endpoint, tableId, model) {
-    var map = App.getLeafletMap();
+    var map = App.getLeafletMap(),
+        ofg = model.get('outlineFeatureGroup');
 
-    if (endpoint && tableId != undefined) {
+    // If the outlines are already present, there is nothing more to do.
+    if (ofg.getLayers().length !== 0) {
```

Comment:
What I meant by "firm understanding of the workflow" is the following: we have an acknowledged bug wherein the shadow mask prevents click events from reaching the UTF layer.  This is supposed to eventually go away (I understand) from changes to the workflow.